src/client/components/App.jsx

```diff
@@ -7,10 +10,20 @@ export const App = () => {
         justifyContent: 'center',
         alignItems: 'center',
         fontSize: 40,
-        color: '#010101'
+        color: '#010101',
       }}
     >
       React homework template
+      <Routes>
+        <Route path="/" />
+        <Route path="/categories/:categoryName" element={<CategoriesPage />} />
+        <Route path="/add" element={<AddRecipiesPage />} />
+        <Route path="/my" element={<MyRecipiesPage />} />
+        <Route path="/favorite" element={<FavoritePage />} />
+        <Route path="/shopping-list" element={<ShoppingListPage />} />
+        <Route path="/search" element={<SearchPage />} />
+        <Route path="*" />
```

Comment:
If you already declared `path="*"` we will use it to navigate our user to page404. 
Add `Page404` element same as in other route's

Reply:
You've done it :)

Reply:
Yes I was in hurry coz Ania code was waiting for your routing.
That's way we implemented it earlier.
I hope you are not angry with it